modules/theme/src/themes/dark/assets/images/identity-providers/facebook.svg

```diff
@@ -18,12 +18,36 @@
  * under the License.
 -->
 
-<svg version="1.1" id="facebook-logo" class="icon" xmlns="http://www.w3.org/2000/svg"
-     xmlns:xlink="http://www.w3.org/1999/xlink" width="100" height="100" viewBox="-15 -15 100 100">
-    <path id="Path_128"
-          d="M60.758,0H8.68A8.688,8.688,0,0,0,0,8.68V60.758a8.688,8.688,0,0,0,8.68,8.68H60.758a8.688,8.688,0,0,0,8.68-8.68V8.68A8.688,8.688,0,0,0,60.758,0Z"
-          fill="#1976d2"/>
-    <path id="Path_129"
-          d="M224.549,117.7H213.7v-8.68c0-2.4,1.944-2.17,4.34-2.17h4.34V96H213.7a13.019,13.019,0,0,0-13.02,13.02v8.68H192v10.85h8.68v23.869H213.7V128.549h6.51Z"
-          transform="translate(-165.961 -82.98)" fill="#fafafa"/>
+<svg id="facebook" class="icon" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"
```

Comment:
Shall we suffix `-idp-illustration` to these ids. Check other places as well.

```suggestion
<svg id="facebook-idp-illustration" class="icon" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"
```

Reply:
Fixed